libcontainerd/types_windows.go

```diff
@@ -67,6 +67,11 @@ type CredentialsOption struct {
 	Credentials string
 }
 
+// DNSOption is a CreateOption that provides container specific DNS settings
```

Comment:
Thanks - yes, I'm trying to figure out how to get rid of side-band options and have everything in OCI.